### DISCUSSION TITLE
'Load more' does nothing for Activities

### DISCUSSION THREAD

**Discussion by habi** [monicahq/monica#6163] [Q&A]
I'm running monica on Heroku.

I added an activity (on my mobile), when I check back on the browser the activity is not shown in the contact but only in the 'Journal'.
Clicking 'load more' in the activities pane on the contact page does nothing.

The issue is that I've seem to have added an activity twice and would like to delete one...

**Comment by ptere** [comment]
Same experience. I see the `GET` request for `/api/contacts/1/activities` is getting a 500, but there's no further detail in the app log.

**Reply by habi** [reply]
I just stumbled over this today again.
After entering an activity with a contact, the activity shows up in the view of the contact, but as soon as I navigate away, the activity is gone.
And also doesn't show up when I click on the 'load more' button anymore.
